phpunit.xml.dist

```diff
@@ -28,4 +28,8 @@
     <logging>
         <junit outputFile="build/report.junit.xml"/>
     </logging>
+    <php>
+        <env name="DB_CONNECTION" value="testing"/>
+        <env name="APP_KEY" value="AckfSECXIvnK5r28GVIWUAxmbBSjTsmF"/>
+    </php>
```

Comment:
Does this also need to be added to every package?

Reply:
Eventually.  The `DB_CONNECTION` declaration eliminates the need to push a bunch of config into the app config on test setup.  The `APP_KEY` needs a definition when making API calls to test Nova resources.